MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,228 @@
+    items.Add(new DataFormMultilineItem() { FieldName = "Address" }); 
+    items.Add(new DataFormAutoCompleteItem() { FieldName = "Country", ItemsSource = contactInfoViewModel.CountryNames }); 
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items;
+
+{% endhighlight}   
+{% endtabs %} 
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel()
+        {
+            this.ContactDetails = new ContactInfo();
+            this.CountryNames = new List<string>
+            {
+                "United States",
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public ContactInfo ContactDetails { get; set; }
+
+        public IList<string> CountryNames { get; set; }
+
+    }
+
+    private IList GetItemSource(string sourceName) 
+    { 
+        var list = new List<string>(); 
+        if (sourceName == "Countries") 
+        { 
+            list.Add("India");
+            list.Add("Japan");
+            list.Add("China");
+            list.Add("USA");
+            list.Add("America");
+            list.Add("UK");
+        } 
+        return list; 
+    }
+{% endhighlight %}
+{% endtabs %}
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class ContactInfo
+    {
+        public string Name { get; set; }
+
+        public string Password { get; set; }
+
+        public string Address { get; set; }
+
+        public string Country { get; set; }
+
+        public string Email { get; set; }
+    }
+
+{% endhighlight %}
+
+### Adding DataForm group manually
+Support has been provided to dynamically add the dataform items to collections using the [Items]`Link` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight XAML %}
+
+    <dataForm:DataFormGroupItem Name="Address">
+                    <dataForm:DataFormGroupItem.Items>
+                        <dataForm:DataFormMultilineItem FieldName="Address"/>
+                        <dataForm:DataFormTextItem FieldName="State"/>
+                        <dataForm:DataFormTextItem FieldName="ZipCode"/>
+                    </dataForm:DataFormGroupItem.Items>
+
+                </dataForm:DataFormGroupItem>
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight C# %}
+
+    DataFormGroupItem groupItem = new DataFormGroupItem();
+        groupItem.Name = "Address";
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Address" });
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "State" });
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "ZipCode" });
+        this.dataForm.Items.Add(groupItem);
+
+{% endhighlight %}
+{% endtabs %}
+
+`AddCustomItemsimages`
+
+### Dynamically remove manually added dataform items
+Support has been provided to dynamically remove the dataform items from collections using the [Items]`Link` property of `SfDataForm`.
```

Comment:
Support has been provided to dynamically remove the dataform items from collections using the [Items]`Link` property of `SfDataForm`.
 
Improve sentence, don't use like this **Support has been provided**

Reply:
content updated